content/blog/2017-11-10-rprofile-mara-averick.md

```diff
@@ -0,0 +1,127 @@
+---
+slug: rprofile-mara-averick
+title: ".rprofile: Mara Averick"
+date: 2017-11-10
+preface: "In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “demystifying the creative and development processes of R community members”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project."
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+categories: blog
+topicid: 921
+tags:
+- R
+- community
+- interviews
+---
+
+<img src="/img/blog-images/2017-11-10-rprofile-mara-averick/mara_averick.png" alt="Mara Averick, Data Nerd At Large" align="left" style="margin: 0px 20px", width="250px">
+
+_Mara Averick is a non-profit data nerd and NBA stats junkie. She is the voice behind two very popular Twitter accounts, @dataandme and @batpigandme. Mara and I discussed sports analytics, how attending a cool conference can change the approach to your career, and how she uses Twitter as a mechanism for self-imposed forced learning._
```

Comment:
pls add Mara's new title here !!

Reply:
based on starting interview text I wonder what's best - maybe refer to fact that interview was done (date) before she got job as dev advocate

Reply:
Just re-read the post and it's going to be so lovely for people to get this insight! Thank you for doing such a great job on these Kelly